packages/component-library/components/Notification/_styles.scss

```diff
@@ -59,7 +61,8 @@ $ca-notification-slide-right: transform $ca-duration-slow ease-out;
 
   &%ca-notification---global {
     width: 100%;
-    padding: $ca-grid / 2 15px;
+    display: flex;
+    align-items: center;
```

Comment:
Is this a nicer way of displaying the same thing, or does this actually change the alignment?